index.js

```diff
@@ -53,6 +53,7 @@ module.exports = (context, options = {}) => {
   let plugins = [
     '@babel/plugin-transform-flow-strip-types', // 2
     ['@babel/plugin-proposal-class-properties', { loose: true }],
+    ['@babel/plugin-proposal-private-methods', { loose: true }],
```

Comment:
Shouldn't we remove them instead ? 

> Thus, you can remove @babel/plugin-proposal-class-properties and @babel/plugin-proposal-private-methods, since they are now enabled by default in @babel/preset-env.

Reply:
That was my first try obviously, but I want to keep the `loose` option only for this plugin, not much choice. 
Also it warns because the `loose` option is different for both plugins, hence the `private-methods` addition...  

I would be in favor of not using the `loose` mode for those anymore (more compiled code), but probably in a major?